tools/run_tests/run_interop_tests.py

```diff
@@ -1477,19 +1465,14 @@ def aggregate_http2_results(stdout):
     http2_server_test_cases = (_HTTP2_SERVER_TEST_CASES
                                if args.http2_server_interop else [])
 
-    report_utils.render_interop_html_report(
-        set([str(l) for l in languages]), servers, _TEST_CASES,
-        _AUTH_TEST_CASES, _HTTP2_TEST_CASES, http2_server_test_cases, resultset,
-        num_failures, args.cloud_to_prod_auth or args.cloud_to_prod,
-        args.prod_servers, args.http2_interop)
-
     if num_failures:
         sys.exit(1)
     else:
         sys.exit(0)
 except Exception as e:
     print('exception occurred:')
     traceback.print_exc(file=sys.stdout)
+    sys.exit(1)
```

Comment:
Why do we want to throw here? Looks like this would affect the cleanup  that happens in the finally block?

Reply:
Kokoro will shows success even if there's an error.

Reply:
Yes, you're right.

Reply:
Got it. I'll do a separate PR.